templates/ehub-existing-setup.json

```diff
@@ -0,0 +1,267 @@
+                "clientAffinityEnabled": false
+            },
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', parameters('Storage Name'))]"
+            ],
+            "resources": [{
+                "apiVersion": "2015-08-01",
+                "name": "web",
+                "type": "sourcecontrols",
+                "dependsOn": [
+                    "[resourceId('Microsoft.Web/Sites', parameters('name'))]"
+                ],
+                "properties": {
+                    "RepoUrl": "[variables('repo')]",
+                    "branch": "[variables('branch')]",
+                    "IsManualIntegration": true
+                }
+            }]
+        },
+        {
+            "type": "Microsoft.Storage/storageAccounts",
+            "name": "[parameters('Storage Name')]",
+            "apiVersion": "2015-06-15",
+            "location": "[variables('location')]",
+            "properties": {
+                "accountType": "Standard_LRS"
+            },
+            "dependsOn": [],
+            "resources": [
+                {
+                    "type": "blobServices/containers",
+                    "apiVersion": "2018-07-01",
+                    "name": "[concat('default/', variables('dlContainerName'))]",
+                    "dependsOn": [
+                        "[parameters('Storage Name')]"
+                    ],
+                    "properties": {
+                        "publicAccess": "None"
+                    }
+                }
+            ]
+        },
+        {
+            "type": "Microsoft.Authorization/roleAssignments",
+            "name": "[variables('roleAssignmentId')]",
+            "apiVersion": "2017-09-01",
+            "properties": {
+                "roleDefinitionId": "[variables('contributorRole')]",
+                "principalId": "[parameters('Service Principal ID')]",
+                "scope": "[variables('resourceGroupId')]"
+            }
+        },
+        {
+            "apiVersion": "2018-05-01",
+            "name": "nestedTemplate",
+            "type": "Microsoft.Resources/deployments",
+            "resourceGroup": "[parameters('Event Hub Resource Group')]",
+            "properties": {
+                "mode": "Incremental",
+                "template": {
+                    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+                    "contentVersion": "1.0.0.0",
+                    "parameters": {},
+                    "variables": {},
+                    "resources": [
+                        {
+                            "type": "Microsoft.Authorization/roleAssignments",
+                            "name": "[guid(parameters('Service Principal ID'), deployment().name)]",
+                            "apiVersion": "2017-09-01",
+                            "properties": {
+                                "roleDefinitionId": "[variables('readerRole')]",
```

Comment:
So, maybe worth describing what we need this for (is there a "description" attribute here or something of that sort?)